charts/brigade-github-app/values.yaml

```diff
@@ -52,3 +52,4 @@ github:
   # Trigger a Check Suite on Pull Requests
   # This will need to be set to true to enable running Check Suites on PRs originating from forks
   checkSuiteOnPR: true
+  # defaultSharedSecret:
```

Comment:
I'm wondering if we can add a further line of context above?  Something describing how this value can be set to the shared webhook secret setup on the corresponding GitHub App and that this value will be inherited/used by any/all Brigade projects it serves, if not set on the project-level... (assuming I have that right 😄 )

Reply:
Sure. On it.

Reply:
Added it in the latest commit.